### PR TITLE
Closes #282: Update font weight for lead class.

### LINKED ISSUE #282: Lead body copy style is not very distinguishable from body text
In QS2, the lead body copy size is close to the h3 text size. It appears much smaller and less distinguishable in QS2, compared to the body text size.

image

Perhaps increase lead body size to closer to h3 in QS?

### PR DESCRIPTION
**Pull request by danahertzberg**
## Motivation
Make lead text style more distinguishable from body text.

Closes #282 

## Solution
Make font weight match the body font (instead of lighter).

Review site: https://review.digital.arizona.edu/arizona-bootstrap/issue/282/docs/2.0/content/typography/#lead

## Impact
Existing sites using the lead font style will see a visual change.

### CODE REVIEW

**Comment by trackleft** [comment]
Before: https://digital.arizona.edu/arizona-bootstrap/docs/2.0/content/typography/#lead
image

After:https://review.digital.arizona.edu/arizona-bootstrap/issue/282/docs/2.0/content/typography/#lead
image

**Review by mmunro-ltrr — DISMISSED** [review]
Merge it now before I break everything! (taking a drastic approach to Hugo failing in GitHub actions).

**Review by trackleft — APPROVED** [review]
Ran it by Tom McCarthy, he had no objections.

**Review by joeparsons — DISMISSED** [review]

**Review by smayden — APPROVED** [review]

File: scss/_custom-styles.scss [hunk 1]
```diff
@@ -103,12 +103,18 @@
...
 .sans {
   font-family: $font-family-sans-serif;
   font-weight: 500;
 }
 
+// >>> Lead
+.lead {
+  font-weight: 400;
+}
+
```
Comment (danahertzberg) [inline scss/_custom-styles.scss:117]:
```suggestion
```

File: scss/_custom-variables.scss [hunk 2]
```diff
@@ -178,6 +178,7 @@ $link-color: $chili;
 
 $heading-margins: 1.042em 0 .667em;
 
+$lead-font-weight: 400;
```
Comment (danahertzberg) [inline scss/_custom-variables.scss:181]:
```suggestion
// fusv-disable
$lead-font-weight: 400;
// fusv-enable
```

**Review by trackleft — APPROVED** [review]

**Comment by trackleft** [comment]
Let's merge this

**Review by smayden — APPROVED** [review]

**Pull request merged**